src/main/java/pageobject/ApplicationOverviewPage.java

```diff
@@ -0,0 +1,17 @@
+package pageobject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class ApplicationOverviewPage extends AbstractPage {
+    private static final By STANDARD_MSF_RATE_FIELD_LOCATOR = By.xpath("//span[text()='* Standard MSF rate']/../../..//*[@data-aura-class='uiOutputPercent']");
```

Comment:
usually LOCATOR is not needed in naming as everybody knows that By is locator

Reply:
It is very good, but I let you discuss it with lecturer.